keyboards/sendyyeah/pix/keymaps/default/keymap.c

```diff
@@ -0,0 +1,226 @@
+#include QMK_KEYBOARD_H
+
+#define LAYERNUM 2
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(KC_MUTE, KC_MPLY, KC_MPRV, KC_MNXT, TO(1)),
+    [1] = LAYOUT(KC_TRNS, KC_MSTP, KC_MRWD, KC_MFFD, TO(0))
+};
+
+int get_icon_start_position(int key_position) {
+    if (key_position == 1) {
+        return 0;
+    } else {
+        return (key_position - 1) * 3;
+    }
+}
+
+void clear_up_down_indicator(void) {
+    wait_ms(3000);
```

Comment:
Just a heads up, this will cause the controller to wait ~3 seconds for this to complete before it does ANYTHING else.

Reply:
think I will remove this method for now, causing some issue on drawing the OLED